index.d.ts

```diff
@@ -7,6 +7,20 @@ export interface Options {
 	@default false
 	*/
 	readonly exact?: boolean;
+
+	/**
+	Allow the domain of an email address to be a local domain (e.g. `user@internal`). Specifically, allow it to have only one dot-separated component.
+
+	@default true
+	*/
+	readonly allowInternalDomain?: boolean;
```

Comment:
Internal domains can have dots, so not entirely correct. Maybe `allowDotlessDomain` or more precise `allowSingleLabelDomain`?